src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -347,31 +347,38 @@ namespace cryptonote
       m_storage_port = command_line::get_arg(vm, arg_sn_bind_port);
 
       bool storage_ok = true;
-
       if (m_storage_port == 0) {
-        MERROR("Please specify the port on which the storage server is listening.");
+        MERROR("Please specify the port on which the storage server with '--" << arg_sn_bind_port.name << " <port>'");
```

Comment:
`Please specify the port on which the storage server *is listening* with...`?

Reply:
Updated